site/content/contribute/getting-started/code-review.md

```diff
@@ -38,47 +38,65 @@ If you are a core committer seeking a review
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `1: PM Review` label only when their review is done.
+        - Product managers ensure the changes meet [user experience guidelines](https://docs.mattermost.com/developer/fx-guidelines.html).
+    * If your changes do not affect the user experience, you may remove `1: PM Review` immediately.
+6. After PM review, request a review from two [core committers](/contribute/getting-started/core-committers/).
+    * The choice of core committers is up to you.
+        - When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
+        - When picking your second core committer, consider someone from another team who may have expertise in the language you're using or the problem you're solving, even if they aren't intricately familiar with the codebase. This can provide a fresh set of eyes on the code to reveal blindspots that are not biased by hitting deadlines, and helps expose the team to new parts of the code to help spread out domain knowledge. This may not make sense for every pull request but is a practice to keep in mind.
+        - Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
+        - Try to avoid assigning the same person to all of your reviews unless they are related.
+        - When in doubt, ask for recommendations on our community server.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `2: Dev Review` label only when these reviews are done.
+        - The last developer to review may choose to help here and speed up the process.
+7. After Dev review, assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers).
+    * The choice of QA tester is up to you.
+        - In most cases, choose the QA tester embedded with your team.
+        - If your change primarily touches another team's codebase, consider their QA tester.
+        - If in doubt, assign any QA tester and comment about the uncertainty. The QA tester may reassign as appropriate.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing.
+        - It is the QA tester's responsibility to determine the scope of required testing, if any.
+    * Remove the `3: QA Review` label only when their review is done.
+        - This is typically the responsibility of the QA tester.
+        - The QA tester may add the `QA Review Done` label when their review is done, or if a review is deemed not necessary.
```

Comment:
Will QA still use `QA Review Done`?

Reply:
My understanding was that this was still under discussion. @lindalumitchell, do you have any new context into the plans for `QA Review Done`? Perhaps we just omit the mention from this document altogether, since it's not applicable to the reviewer.

Reply:
Thanks, yes, we are still finalizing decisions about the labels we will use for visibility, alerting, and reporting on the QA side. I think it is fine to remove the line about `QA Review Done` entirely from this doc, and we can include it in QA-specific docs as needed.  @hanzei @lieut-data

Reply:
Oh, what about the `4: Reviews Complete` label; still using that one, right? Useful to include a mention here, maybe something like "The QA tester adds the label `4: Reviews Complete` when all reviews are done." to replace the line about `QA Review Done`?

Reply:
I admit I'm not keen on the `4: Reviews Complete` label. We haven't used it consistently in the past, and it effectively duplicates information already present in the pull request between the requested reviews and blocking labels.

Thoughts on just omitting it altogether?

Reply:
To me `4: Reviews Complete` serves as a "green light" from QA that all necessary reviews had happend. But I see that it's cumbersome work and it's not used consistently.

I'm fine with dropping it. Maybe we don't remove the label (jet) so we can return to the label if needed.

Reply:
Sounds good! I'll put `4: Reviews Complete` back into the document.